scss/style.scss

```diff
@@ -1,138 +1,147 @@
+$cadetBlue: #429EA6;
+$indigoDye: #153B50;
+$alabaster: #ECEBE4;
+
 body {
     font-family: 'Open Sans', sans-serif;
-    background-color: #f0ecec;
     padding-top: 50px;
 }
 
-.calculator {
-    
+.calculator-container {
     width: 330px;
     height: 430px;
-    background-color: #797979;
+    background-color: $alabaster;
     margin: 0 auto;
-    border-radius: 10px 10px 5px 5px;
-    border: 3px solid rgb(30, 4, 77);
+    padding: 0px 10px;
```

Comment:
Please, add top padding to calculator. I think it will be nicer then,

![Untitled](https://user-images.githubusercontent.com/13486264/128594188-9e50298c-1bc1-41bf-9922-91b8cabac921.png)